src/api/pubsub.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const bs58 = require('bs58')
+var Base64 = require('js-base64').Base64
+var Stream = require('stream')
+var http = require('http')
+
+let activeSubscriptions = []
+
```

Comment:
Looking at what activeSubscriptions does, would it make sense to put them in a map?

```javascript
// add
activeSubscriptions[topic] = {}

// check
Object.keys(activeSubscriptions).include(topic)

// remove
delete activeSubscriptions[topic]
```